app/commands/v2/put_content.rb

```diff
@@ -77,6 +78,16 @@ def set_timestamps(edition)
         Edition::Timestamps.edited(edition, payload, previously_published_edition)
       end
 
+      def set_publishing_user(edition)
+        return unless payload[:last_edited_by]
+
+        edition.author = Author.find_or_create_by(
+          name: payload[:last_edited_by][:name],
+          email: payload[:last_edited_by][:email],
```

Comment:
**suggestion:** multiple access

Is it worth assigning `payload[:last_edited_by]` to a variable before line 82, since it's accessed three times if present? We did similar in `create_content_id_alias`